src/screens/station_info_screen.java

```diff
@@ -0,0 +1,88 @@
+package screens;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+import game_objects.game_manger;
+import helpers.C.GUIConsoleIO;
+import helpers.gconsole_menu;
+import helpers.gyinput;
+import game_objects.item;
+
+/*station market terminal
+ * where the player can buy and sell goods
+ * */
```

Comment:
if you copy code make sure to change relvant coments

Reply:
Right, will fix later